### DISCUSSION TITLE
How do I know if I am using autofree memory management

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#16732] [General]
I don't like GC, but rust is too complicated for me to learn. Vlang supports non-GC automatic memory management, but how do I know when I use Blom-GC or autofree?

**Comment by SirZenith** [comment]
It says in the documentation that autofree is still work in progress, and we should avoid using it for now. When it's ready, it will come as default.

**Reply by JalonSolov** [reply]
You can try it if you like, by adding `-autofree` switch to the command line.  Simply be aware that it is not complete, yet.
